app/javascript/addItem.js

```diff
@@ -1,18 +1,22 @@
 let images = [];
 let container = document.getElementById("upload");
 
+
 window.loadFiles = function(event) {
   const pictures = event.target.files;
+  console.log(pictures);
   for(let i=0; i<pictures.length; i++) {
     images.push(pictures[i]);
   }
+  console.log(images);
+  console.log(event.target.files);
```

Comment:
This `console.logs` are necessary in the code?

Reply:
No, a mistake let them in the code. Fixed!